Sandbox/Sandbox/Info.plist

```diff
@@ -45,7 +45,7 @@
 			<string>Editor</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>reachfive</string>
+				<string>reachfive-TYAIHFRJ2a1FGJ1T8pKD</string>
```

Comment:
Maybe there's a way of getting this value automatically in the SDK from the plist. The doc [here](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102207) says there's also a `CFBundleURLName` key available, so we could perhaps have a value like `reachfiveUrlScheme` for this and load it from the code ? This way it would avoid having to duplicate the value in two places (the plist + the app code when calling the SDK) ?

I'm not familiar with IOS dev so that's just a thought after having browsed the docs for 10 minutes. Please disregard if it's a stupid idea (and there's a high chance it is.)

Reply:
Yes there are ways to retrieve plist values in SDK, but is that what we want to do?
I don't wanna radically change the way `SdkConfig` object is instancied, si adding a new value `redirectUri` seems OK to me.